tests/Integration/BUILD.bazel

```diff
@@ -598,7 +598,7 @@ php_gapic_library(
     name = "spanner_php_gapic",
     migration_mode = "MIGRATING",
     srcs = ["@com_google_googleapis//google/spanner/admin/database/v1:database_proto_with_info"],
-    service_yaml = "@com_google_googleapis//google/spanner/admin/database/v1:spanner_admin_database.yaml",
+    service_yaml = "@com_google_googleapis//google/spanner/admin/database/v1:spanner_admin_database_gapic.yaml",
```

Comment:
service yaml and gapic yaml are different things, you'll want to use [`spanner.yaml` instead](https://github.com/googleapis/googleapis/blob/master/google/spanner/admin/database/v1/spanner.yaml)

Reply:
Thank you for this information, I was confused on what to actually add, I will change it right away.

Reply:
Changed. Thank you!